app/uk/gov/openregister/StreamUtils.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.openregister;
+
+import java.util.Iterator;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+public class StreamUtils {
```

Comment:
This is used in one single place. Can you not use 'StreamSupport.stream()' directly?